packages/app/src/components-styled/line-chart/index.tsx

```diff
@@ -88,9 +88,17 @@ export function LineChart<T extends Value>({
   isPercentage,
   showMarkerLine = false,
   formatMarkerLabel,
-  padding = defaultPadding,
+  padding = {},
   showLegend = false,
 }: LineChartProps<T>) {
+  const chartPadding: ChartPadding = useMemo(
+    () => ({
+      ...defaultPadding,
+      ...padding,
+    }),
+    [padding]
+  );
```

Comment:
I'd opt for renaming this const to `padding` and rename the incoming prop to something like paddingOverride/overridePadding .. eg:

```suggestion
  padding: overridePadding = {},
  showLegend = false,
}: LineChartProps<T>) {
  const padding: ChartPadding = useMemo(
    () => ({
      ...defaultPadding,
      ...overridePadding,
    }),
    [overridePadding]
  );
```